### PR TITLE
test: add tests for bitbucket indexer

### PR DESCRIPTION
**Pull request by balins**
[FREGE-163](https://jira.frege.ii.uj.edu.pl/browse/FREGE-163)

### CODE REVIEW

**Comment by DawidSzczerba** [comment]
Maybe it would be better to have the following files structure fregepoc/indexers/tests/bitbucket ?

**Comment by balins** [comment]
@DawidSzczerba How would this structure be better? Because I don't see any advantage of this additional nesting.

**Comment by DawidSzczerba** [comment]
Discussed with @balins offline. For me everything is ok, you can merge and I will improve it in my file structure. For sourceforge there will be a bit more files with tests, because this indexer differs significantly from others.

File: backend/fregepoc/indexers/tests/test_bitbucket.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+from pathlib import Path
+
+import pytest
+import requests
+from django.conf import settings
+from pytest_mock import MockerFixture
+
+from fregepoc.indexers.models import BitbucketIndexer
+
+MOCK_PATH = settings.BASE_DIR / "fregepoc" / "indexers" / "tests" / "__mock__"
```
Comment (DawidSzczerba) [inline backend/fregepoc/indexers/tests/test_bitbucket.py:11]: I think you can move this constant variable to a separate file (maybe constants.py), I use this in my tests too.

**Review by DawidSzczerba — APPROVED** [review]

File: backend/fregepoc/indexers/tests/test_bitbucket.py (hunk 1 above)
Comment (balins) [reply backend/fregepoc/indexers/tests/test_bitbucket.py]: refactored in 909d8fd

**Pull request merged**